src/explore-education-statistics-frontend/src/modules/table-tool/components/__tests__/TableToolFinalStep.test.tsx

```diff
@@ -305,10 +305,10 @@ describe('TableToolFinalStep', () => {
         contactTelNo: '012345',
       },
       methodology: {
-        id: '',
-        slug: '',
-        summary: '',
-        title: '',
+        id: 'm1',
+        slug: 'm1',
+        summary: 'words',
+        title: 'methodology title',
```

Comment:
Would suggest adding some additional assertions (besides the snapshot) to test that the methodology link renders as expected